neqo-qpack/src/encoder.rs

```diff
@@ -41,10 +42,11 @@ pub struct QPackEncoder {
 
 impl QPackEncoder {
     #[must_use]
-    pub fn new(use_huffman: bool) -> Self {
+    pub fn new(max_table_size: u64, use_huffman: bool) -> Self {
```

Comment:
You could make this take an argument of `QpackSettings`, I think.  That there are values this doesn't use isn't that big of a deal.